tmt/utils.py

```diff
@@ -1547,15 +1555,24 @@ def render_exception(exception: BaseException) -> List[str]:
             *render_run_exception(exception)
             ]
 
+    if 'TMT_SHOW_TRACEBACK' in os.environ and os.environ['TMT_SHOW_TRACEBACK'] != '0':
```

Comment:
Maybe with get the line would be shorter?

```suggestion
    if os.environ.get('TMT_SHOW_TRACEBACK') != '0':
```

Reply:
Fixed in e42f5708. I think it cannot be `os.environ.get('TMT_SHOW_TRACEBACK')` without a default value, because when not set, we'd get `None` & emit the traceback.